_sass/0-base/_reset.scss

```diff
@@ -0,0 +1,134 @@
+// http://meyerweb.com/eric/tools/css/reset/
```

Comment:
Just wanna verify we’re minifying/gzipping in the build phase because helllllo lines 😅

Reply:
I'm minifying based on the Sass yaml config options
<img width="728" alt="Screen Shot 2020-06-12 at 4 16 50 PM" src="https://user-images.githubusercontent.com/1668411/84546976-286ead00-acc8-11ea-894a-67a5e98d1caa.png">